recipes-wpe/wpeframework/include/cobalt.inc

```diff
@@ -21,5 +21,5 @@ PACKAGECONFIG[cobalt] = " \
 "
 # ----------------------------------------------------------------------------
 
-DEPENDS += "libcobalt"
-RDEPENDS_${PN} += "libcobalt"
+DEPENDS += " libcobalt"
+RDEPENDS_${PN} += " libcobalt"
```

Comment:
It looks like the header files need to be installed and packaged in the libcobalt recipe.